kamon-logback-stackdriver/src/main/scala/kamon/stackdriver/JsonStringBuilder.scala

```diff
@@ -36,6 +36,11 @@ private[stackdriver] final class JsonStringBuilder(val underlying: java.lang.Str
     this
   }
 
+  def appendString(s: String): JsonStringBuilder = {
```

Comment:
I would remove that

Reply:
but we need to simply append these `null` or `true` etc. to the underlying builder

Reply:
so I can add `encodeNull` and `encodeBoolean` ?